locales/templates/app.pot

```diff
@@ -2384,3 +2423,48 @@ msgctxt "travel_search_engine_fxlite"
 msgid "See results in %1$s Travel Discovery"
 msgstr ""
 
+#. Dialog
+#. dialog title for introducing Firefox Lite Travel Discovery
+msgctxt "travel_dialog_1_title"
+msgid "Discover More!"
+msgstr ""
+
+#. dialog description for introducing the feature. The %1$s here is Firefox
+#. Lite.
+#, c-format
+msgctxt "travel_dialog_1_description"
+msgid "Explore things to do, accomodations, and more in %1$s Travel Discovery."
+msgstr ""
+
+#. Action button for seeing the search result in Firefox Lite Travel Discovery
+msgctxt "travel_dialog_1_action_1"
+msgid "TRY TRAVEL DISCOVERY"
+msgstr ""
+
+#. Action button for dismissing this prompt
+msgctxt "travel_dialog_1_action_2"
+msgid "NO, THANKS"
+msgstr ""
+
+#. dialog title for asking if users want to change default search engine
```

Comment:
Comment doesn't correspond to  string

Reply:
Do you mean "Action button for dismissing this prompt"?
The prompt here referred to the dialog mentioned above, so one action button is to try travel discovery, and the other action button is to dismiss the prompt, does that make sense?

Reply:
I rewrote the comment and hope it becomes clearer for you.